src/doc/index.md

```diff
@@ -6,32 +6,44 @@ nav {
 }
 </style>
 
-This is an index of the documentation included with the Rust
-compiler. For more comprehensive documentation see [the
-website](https://www.rust-lang.org).
+This page is an overview of the documentation included with your Rust install.
+Other unofficial documentation may exist elsewhere; for example, the [Rust
+Learning] project collects documentation from the community, and [Docs.rs]
+builds documentation for individual Rust packages.
 
-[**The Rust Programming Language**][book]. Also known as "The Book",
-The Rust Programming Language is the most comprehensive resource for
-all topics related to Rust, and is the primary official document of
-the language.
+## API Documentation
 
-[**The Rust Reference**][ref]. While Rust does not have a
-specification, the reference tries to describe its working in
-detail. It is accurate, but not necessarily complete.
+Rust provides a standard library with a number of features; you can browse
+its documentation [here][api].
 
-[**Standard Library API Reference**][api]. Documentation for the
-standard library.
+## Reference Documentation
 
-[**The Rustonomicon**][nomicon]. An entire book dedicated to
-explaining how to write unsafe Rust code. It is for advanced Rust
-programmers.
+Rust does not yet have a formal specification, but we have a reference document
+[ref][ref]. It is guaranteed to be accurate, but not complete. We now have a
+policy that all new features must be included in the reference before
+stabilization; however, we are still back-filling things that landed before
+then. That work is being tracked [here][38643].
 
-[**Compiler Error Index**][err]. Extended explanations of
-the errors produced by the Rust compiler.
+## Extended Error Documentation
 
-[book]: book/index.html
-[ref]: reference.html
+All of Rust's errors code with error codes, and you can request extended
```

Comment:
This isn't true though, there are errors which don't have error codes like `error: main function not found`.